package.json

```diff
@@ -30,13 +30,14 @@
     "update-repos": "node src/utilities/fetch-package-repos.js",
     "content": "node src/scripts/build-content-tree.js ./src/content ./src/_content.json",
     "build-test": "npm run build && http-server dist/",
+    "bundle-analyze": "run-s clean fetch content && cross-env NODE_ENV=production webpack --config webpack.ssg.js && cross-env NODE_ENV=production webpack --config webpack.prod.js  --profile --json > stats.json && webpack-bundle-analyzer stats.json",
```

Comment:
Didnt want to hook --profile onto our prod builds to not slow down CI on prs

Reply:
Yeah, it would be a post build command